src/AbstractBench.php

```diff
@@ -51,4 +51,6 @@ public function provideComplexity(): array
     abstract public function initSerializer(): void;
 
     abstract protected function serialize(Forum $data): void;
-}
\ No newline at end of file
+
+    abstract public function getPackageName(): ?string;
```

Comment:
What about `abstract public function getPackageName(): string;`

(not nullable)

Reply:
Why not `getName`, as you see there are some tests that are not a composer package

Reply:
The first version of this PR had the method `getName` but was removed after @bendavies [comments](https://github.com/php-serializers/ivory-serializer-benchmark/pull/13#issuecomment-439638500)